modules/cloud/modules/start/pages/overview.adoc

```diff
@@ -10,9 +10,14 @@ handling all the complexity of deploying and managing your deployments on the cl
 
 [NOTE]
 ====
-TigerGraph Cloud 4.0 (beta version) was released on Apr. 29th, 2024.
+TigerGraph Cloud 4 was released on Oct. 30th, 2024.
```

Comment:
```suggestion
TigerGraph Cloud 4 was released on Oct. 30, 2024.
```
This is a cloud classic page. Are you sure you want the Cloud 4 release on this page?

Reply:
Yes, because we want to let the cloud v3 users know that v4 is out, and v3 is going to be retired soon.